tests/dummy/app/templates/components/materialize-icon.hbs

```diff
@@ -1,3 +1,5 @@
 {{!-- BEGIN-SNIPPET materialize-icon --}}
-<i class="lt-sort-icon material-icons">{{get @sortIcons @sortIconProperty}}</i>
+<span>
```

Comment:
Why did we wrap this in a span?

Reply:
The component was using `tagname: 'span'`